apps/web/src/components/task/task-activities.tsx

```diff
@@ -38,22 +40,43 @@ function TaskActivities() {
               )}
             </div>
           </div>
-          <div className="flex-1">
+          <div className="flex-1 flex flex-col gap-2">
             <div className="flex items-center gap-2">
-              <span className="text-sm font-medium text-zinc-900 dark:text-zinc-100">
+              <span className="text-sm font-medium text-zinc-900 dark:text-zinc-400">
                 {activity.userEmail}
               </span>
               <span className="text-xs text-zinc-500 dark:text-zinc-400">
                 {formatDistanceToNow(activity.createdAt)} ago
               </span>
             </div>
-            <div className="mt-1.5">
-              <div className="text-sm text-zinc-600 dark:text-zinc-400">
+            <div className="flex flex-col gap-1">
+              <div className="text-sm text-zinc-600 dark:text-zinc-100">
                 {activity.type === "create" && activity.content}
                 {activity.type === "update" && activity.content}
                 {activity.type === "status" && activity.content}
                 {activity.type === "comment" && activity.content}
               </div>
+              <div className="flex flex-row gap-3">
+                <button
+                  type="button"
+                  className="text-xs underline text-zinc-500 dark:text-zinc-400"
+                >
+                  Edit
+                </button>
+                <button
+                  className="text-xs underline text-zinc-500 dark:text-zinc-400"
+                  tabIndex={0}
+                  type="button"
+                  onClick={() => deleteComment({ id: activity.id })}
+                  onKeyDown={(e) => {
+                    if (e.key === "Enter" || e.key === " ") {
+                      deleteComment({ id: activity.id });
+                    }
```

Comment:
See if there is no utils method to handle this

Reply:
There are no utils for this